gst/tensor_filter/tensor_filter.c

```diff
@@ -156,6 +156,7 @@ enum
   PROP_OUTPUT,
   PROP_OUTPUTTYPE,
   PROP_OUTPUTNAME,
+  PROP_NNAPI,
```

Comment:
Do NOT add subplugin-specific item to global data structure.

Reply:
Ok, seems tensor_filter represents other frameworks. However, I think only way to pass a property to other frameworks is though tensor_filter as of now. How could I pass the framework specific property to tensorflow lite module, tensor_filter_tensorflow_lite_core?

Reply:
As discussed a few hours ago, use PROP_CUSTOM to get the values.